sw/device/silicon_creator/mask_rom/docs/_index.md

```diff
@@ -118,18 +116,14 @@ Dependencies:
 *   Entropy?
 *   Clocks
 
-## Lockdown
+## Memory Protection
```

Comment:
Is everyone happy with this rename? We could keep it as 'lockdown' to avoid further confusion. I originally tried to rename it to 'unlock address space' module but that name didn't really stick.